src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -576,6 +576,21 @@ public function compileRandom($seed)
     }
 
     /**
+     * Compile the field database function into SQL.
+     * @param string $column
+     * @param array $columnValues
+     * @return string
+     */
+    public function compileField($column, $columnValues)
+    {
+        $columnValues = collect($columnValues)->map(function ($columnValue) {
+            return '"'.$columnValue.'"';
+        })->implode(', ');
+
+        return 'field(`'.$column.'`, '.$columnValues.')';
```

Comment:
This will not work in all cases, lets say you join two tables
```SQL
SELECT users.name, posts.title,posts.priority FROM users JOIN posts ON users.id = posts.user_id ORDER BY FIELD(posts.title,"IMPORTANT","NORMAL","SECONDARY")
```

Your code will result in:
```SQL
SELECT users.name, posts.title,posts.priority FROM users JOIN posts ON users.id = posts.user_id ORDER BY FIELD(`posts.title`,"IMPORTANT","NORMAL","SECONDARY")
```
Which is not valid.

Reply:
Hi,

I appreciate the feedback. I was really trying to see if people are interested in such a proposal. I am well-aware of the problems that may appear with the current code. 

Since, there's some interest, I will continue working on it :)